### PR TITLE
Add meetings and projects to Analyzers SIG README

### PR DESCRIPTION
**Pull request by pranavgaikwad**

### CODE REVIEW

File: sig-analyzers/README.md [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+
+- Analyzers SIG meeting (Bi-weekly): Thursdays at 11:30 ET
+  - [Video Call](https://meet.google.com/siu-iewh-srw)
+  - [Meeting notes and Agenda](https://docs.google.com/document/d/1du14zVpdysSGNvYwMJ_UZZme3ojQIim58v2NWWa0kpc)
 
 ## Contact
 
 - Slack: #konveyor
-- Mailing List: [konveyor-analyzers@googlegroups.com](https://groups.google.com/u/1/g/konveyor-analyzers)
+- Mailing List: [konveyor-analyzers@googlegroups.com](https://groups.google.com/u/1/g/konveyor-analyzers
```
Comment (savitharaghunathan) [inline sig-analyzers/README.md:16]: nit: missing ')' at the end

File: sig-analyzers/README.md [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+
+## Projects
+
+- [analyzer-lsp](https://github.com/konveyor/analyzer-lsp): Core analyzers developed by SIG based on Language Server Protocol.
+
+- [enhancements](https://github.com/konveyor/enhancements): Design proposals to communicate changes to the core analyzers. 
 
+- [GitHub Project][https://github.com/orgs/konveyor/projects/46]: GitHub project to track ongoing & future work in core analyzers.
```
Comment (savitharaghunathan) [inline sig-analyzers/README.md:24]: nit: link paran, switch to '( )' to display as link

**Review by savitharaghunathan — COMMENTED** [review]
few nits, otherwise 
/lgtm

**Comment by pranavgaikwad** [comment]
@savitharaghunathan thank you for catching my stupid mistakes :) fixed now

**Review by savitharaghunathan — APPROVED** [review]
/lgtm

**Pull request merged**